rllib/utils/framework.py

```diff
@@ -108,6 +109,24 @@ def try_import_tf(error: bool = False):
     return tf1_module, tf_module, version
 
 
+# Fake module for tf.
+class _TFStub:
+    def __init__(self) -> None:
+        self.keras = _KerasStub()
+
+
+# Fake module for tf.keras.
+class _KerasStub:
+    def __init__(self) -> None:
+        self.Model = _FakeTfClassStub
+
+
+# Fake classes under keras (e.g for tf.keras.Model)
+class _FakeTfClassStub:
```

Comment:
should these go into some test util module or here?
just checking.

Reply:
We can leave it here for now. The location is implementation details. won't break anything if we change it later.